### PR TITLE
Add meditation story

### PR DESCRIPTION
**Pull request by edewit2**

### CODE REVIEW

**Comment by edewit2** [comment]
@casey-collab

File: english/meditation/meditation.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+You should relax, you completed the git course, good job!
```
Comment (casey-collab) [inline english/meditation/meditation.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by edewit2** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Looks great! :star:

**Pull request merged**